src/conf/extensions.yml

```diff
@@ -31,3 +31,7 @@ extensions:
   folders:
     - che-theia-activity-tracker
   checkoutTo: master
+- source: https://github.com/eclipse/che-theia
+  folders:
+    - extensions/eclipse-che-theia-plugin-ext
```

Comment:
we only need one folder ?
I thought we wanted as well `extensions/eclipse-che-theia-plugin` (to avoid some wired copy of files between two modules)

Reply:
No, we don't need that, as `eclipse-che-theia-plugin` published on npm as `@eclipse-che/plugin` and `extensions/eclipse-che-theia-plugin-ext` depends on it.

Reply:
@evidolob I think it's not helping the developer workflow to not include the module
if I modify the module locally, I want to be able to test it probably without publishing it first

Reply:
OK, I will cover that flow